### DISCUSSION TITLE
Sigo sin poder instalar el compilador

### DISCUSSION THREAD

**Discussion by ELVAVAS** [INGCOM-UNRN-P1/cursada-2023#15] [Preguntas y respuestas] [answered]
He hecho lo que dice la wiki, pero cada vez reinicio o cierro sesion, todo se borra del path. tengo la carpeta en esa direccion y luego el cmd no reconoce el último texto

**Comment by TeoInMaghine** [comment]
Cuando ponés el path está exactamente igual que en la wiki? Importante que barra normal ( / ) y barra invertida ( \\ ) no son lo mismo.

**Reply by ELVAVAS** [reply]
Sí, está exactamente igual

**Comment by martinvilu** [comment]
Como estas configurando el `path`?

**Reply by ELVAVAS** [reply]
Como lo muestra en la wiki. El punto es que ahora funciona pero cada vez que reinicio la computadora lo tengo que volver a hacer porque el path se borra

**Comment by Agusbruzzese** [comment]
![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/96895097/3e62194c-eee9-4a09-8227-fda0d45e528f)
si vas ahi, pones "enviroment Variables", dentro de system variables elegis path, y "edith" y agregas el pat de min usualmente (C:\Minw\bin)

**Reply by ELVAVAS** [reply]
Sí, eso hice, el problema que tenía es que lo que le agrego al path se borra cuando apago o cierro sesión, ahora cuando acabe la clase voy a reintentar a ver si deja de pasar

**Reply by Agusbruzzese** [reply]
tu maquina tiene alguna restriccion en el uso? o tu usuario?

**Reply by Agusbruzzese** [reply, marked as answer]
El Editor del Registro de Windows 10 no guarda mis cambios

    Presione la tecla del logotipo de Windows + R en su teclado> escriba regeditar en el cuadro Ejecutar para abrir el Editor de registrolocalizar enlace navegador windows 7 Outlook 2003
    si obtienes un Control de cuentas del usuario cuadro de diálogo, seleccione DE ACUERDO
    Abra la ubicación de la clave que desea cambiar en el Editor del Registro
    Haga clic derecho en la tecla> abrir permisos…
    Seleccione Avanzado
    Si tu usuario no es el propietario, elige Cambiar
    Escriba su nombre de usuario en el Ingrese el nombre del objeto para seleccionar caja > hacer clic Comprobar nombres > prensa DE ACUERDO
    Ahora debería mostrar su nombre como propietario
    Si también desea tomar posesión de las subclaves, marque la casilla junto a Reemplazar propietario en subcontenedores y objetos > Seleccione Solicitar
    presione el Deshabilitar herencia botón si está disponible
    Hacer clic Agregar
    Hacer clic Seleccione un director en el Entrada de permiso ventana
    En el Ingrese el nombre del objeto para seleccionaringresa tu nombre > selecciona Comprobar nombres > hacer clic DE ACUERDO
    En el Entrada de permiso ventana debajo Permiso básicomarque la casilla junto a Control total > hacer clic DE ACUERDO
    Si desea aplicar las mismas reglas de permisos a todas las subclaves, marque la casilla junto a Reemplace todos los permisos de objetos secundarios con permisos heredables de este objeto > hacer clic DE ACUERDO
    Si luego decide quitar su permiso, simplemente desmarque la casilla Control total caja o eliminar su nombre de la Nombres de grupos o usuarios caja.

**Reply by Agusbruzzese** [reply]
https://expertosenlinea.com.ar/que-hacer-si-regedit-no-guarda-los-cambios-full-guide/?expand_article=1 eso es para revisar los permisos del regedit

**Reply by ELVAVAS** [reply]
funcionó!!! Muchisimas gracias. Hasta me entero que habia algo raro con mi usuario
